core/src/blog/gamedevelopmentbox2dtutorial/views/MenuScreen.java

```diff
@@ -69,11 +69,11 @@ public void show() {
 
         // Add buttons to table
         table.add(newGame).fillX().uniformX();
-        table.row().padTop(10);
+        table.row().padTop(50);
```

Comment:
Should maybe use a variable for the pad instead of having to change each of the values 50. In my branch i had another button and that button-pad is still 10. So i had to manually change it to 50 when merging.